<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWidget</class>
 <widget class="QWidget" name="MainWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>730</width>
    <height>606</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QComboBox" name="stateCombo"/>
     </item>
     <item>
      <widget class="QComboBox" name="typeCombo">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>260</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="whatsThis">
        <string>Update Types</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KLineEdit" name="searchLine">
       <property name="whatsThis">
        <string>Search Bar</string>
       </property>
       <property name="clickMessage">
        <string>Click to search</string>
       </property>
       <property name="showClearButton" stdset="0">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="searchButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="whatsThis">
        <string>Search Button</string>
       </property>
       <property name="text">
        <string>Search</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Sort by :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="sortCombo">
         <item>
          <property name="text">
           <string>Popularity</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Download Size</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Install Size</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="GroupList" name="groupList">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>230</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>260</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="whatsThis">
      <string>Category Panel</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="PackageView" name="packageList">
     <property name="minimumSize">
      <size>
       <width>450</width>
       <height>0</height>
      </size>
     </property>
     <property name="whatsThis">
      <string>Package List</string>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="showDropIndicator" stdset="0">
      <bool>false</bool>
     </property>
     <property name="dragDropOverwriteMode">
      <bool>false</bool>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::NoSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="showGrid">
      <bool>false</bool>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <property name="cornerButtonEnabled">
      <bool>false</bool>
     </property>
     <attribute name="horizontalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderDefaultSectionSize">
      <number>52</number>
     </attribute>
     <attribute name="verticalHeaderMinimumSectionSize">
      <number>52</number>
     </attribute>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QWidget" name="actions" native="true">
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>4</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>4</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="busyIndicator">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="KSqueezedTextLabel" name="statusLabel">
        <property name="scaledContents">
         <bool>true</bool>
        </property>
        <property name="indent">
         <number>4</number>
        </property>
        <property name="textElideMode">
         <enum>Qt::ElideRight</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="KPushButton" name="cancelButton">
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="KPushButton" name="actionButton">
        <property name="text">
         <string>Install Package(s)</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KSqueezedTextLabel</class>
   <extends>QLabel</extends>
   <header>ksqueezedtextlabel.h</header>
  </customwidget>
  <customwidget>
   <class>KPushButton</class>
   <extends>QPushButton</extends>
   <header>kpushbutton.h</header>
  </customwidget>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
  <customwidget>
   <class>PackageView</class>
   <extends>QTableView</extends>
   <header location="global">packageview.h</header>
  </customwidget>
  <customwidget>
   <class>GroupList</class>
   <extends>QListWidget</extends>
   <header location="global">grouplist.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../data/data.qrc"/>
 </resources>
 <connections/>
</ui>
